chromadb/api/fastapi.py

```diff
@@ -250,9 +253,9 @@ def create_collection(
     @override
     def get_collection(
         self,
-        name: Optional[str] = None,
+        name: str,
         id: Optional[UUID] = None,
-        embedding_function: Optional[EmbeddingFunction] = ef.DefaultEmbeddingFunction(),
+        embedding_function: Optional[EmbeddingFunction[Embeddable]] = ef.DefaultEmbeddingFunction(),  # type: ignore
```

Comment:
why is this ignore needed?

Reply:
mypi doesn't handle `Protocol` subtypes well, so doesn't recognize that in this context, `Document` is a subtype of `Embeddable`. 

This is a known mypi issue.